File: 3194-find-words-containing-character/find-words-containing-character.go
func findWordsContaining(words []string, x byte) []int {
    res:=[]int{}
    for i,ele := range words {
        for j:=0;j<len(ele);j++{
            if x == ele[j] {
                res = append(res,i)
                break
            }
        }
    }
    return res
}

File: 3194-find-words-containing-character/find-words-containing-character.java
class Solution {
    public List<Integer> findWordsContaining(String[] words, char x) {
        ArrayList<Integer> res = new ArrayList<>();
        for(int i=0;i<words.length;i++){
            for(char c:words[i].toCharArray()){
                if (x == c){
                    res.add(i);
                    break;
                }
            }
        }
        return res;
    }
}